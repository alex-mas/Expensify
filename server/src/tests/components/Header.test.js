import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Header from '../../components/header';


describe('Header component unit tests', function(){
    test('should render header correctly',()=>{
        const wrapper = shallow(<Header/>);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('h1').length).toBe(1);
        expect(wrapper.find('h1').text()).toBe('Expensify App');
        /*const renderer = new ReactShallowRenderer();
        renderer.render(<Header/>);
        expect(renderer.getRenderOutput()).toMatchSnapshot();*/
    });
});
