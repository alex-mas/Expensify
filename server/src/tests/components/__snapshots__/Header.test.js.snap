// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component unit tests should render header correctly 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
     Home
  </NavLink>
  |
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/create"
  >
     Create
  </NavLink>
  |
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/help"
  >
     Help
  </NavLink>
  |
</header>
`;
